utils/get_supported_scylla_base_versions.py

```diff
@@ -14,7 +14,7 @@
 
 
 # We support to migrate from specific OSS version to enterprise
-supported_src_oss = {'2021.1': '4.3', '2022.1': '5.0', '2022.2': '5.1', '2023.1': '5.2', '2023.2': '5.3'}
+supported_src_oss = {'2021.1': '4.3', '2022.1': '5.0', '2022.2': '5.1', '2023.1': '5.2', '2024.1': '5.4'}
```

Comment:
we no longer support 5.3->2023.2? while still from 4.3 to 2021.1?

Reply:
2023.2 doesn't exists (and not going to be)
5.3 was canceled

Reply:
true... :smile: